tests/golden/test_mapper_maxcut_last.qasm

```diff
@@ -1,4 +1,4 @@
-# Generated by OpenQL 0.10.0 for program test_mapper_maxcut
+# Generated by OpenQL 0.10.5 for program test_mapper_maxcut
 version 1.2
```

Comment:
This seems to be a different circuit. Where are the blue and red gates. I see move_init has disappeared to also the options have changed!